stix_shifter_modules/guardium/stix_translation/json/guardium_qsearch_params_map.json

```diff
@@ -12,7 +12,7 @@
     "ServerIP": ["Policy Violation:ATA Open Cases"],
     "Severity": ["Policy Violation:ATA Open Cases"],
     "OS User": ["Policy Violation:ATA Open Cases"],
-    "Client IP": ["Policy Violation:ATA Open Cases"]
+    "ClientIP": ["Policy Violation:ATA Open Cases"]
```

Comment:
have you verified if the field name change `Client IP` works with the datasource?